### PR TITLE
GH-531: Support tombstones on output

### PR DESCRIPTION
**Pull request by aldex32**
resolves https://github.com/spring-cloud/spring-cloud-stream-binder-kafka/issues/531

### CODE REVIEW

File: spring-cloud-stream-binder-kafka/src/test/java/org/springframework/cloud/stream/binder/kafka/integration/KafkaNullConverterTest.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+	private static CountDownLatch countDownLatch = new CountDownLatch(1);
+
+	@Autowired
+	private MessageChannel output;
+
+	@Test
+	public void testKafkaNullConverter() throws InterruptedException {
+		output.send(new GenericMessage<>(KafkaNull.INSTANCE));
+
+		assertThat(countDownLatch.await(500, TimeUnit.MILLISECONDS)).isTrue();
```
Comment (garyrussell) [inline spring-cloud-stream-binder-kafka/src/test/java/org/springframework/cloud/stream/binder/kafka/integration/KafkaNullConverterTest.java]: I suggest a larger wait time here; sometimes the CI build servers are busy and we'll likely get a false failure here.

10 seconds is what we usually use.

Comment (aldex32) [reply spring-cloud-stream-binder-kafka/src/test/java/org/springframework/cloud/stream/binder/kafka/integration/KafkaNullConverterTest.java]: fixed!

**Comment by garyrussell** [comment]
I didn't notice that the test used a `@KafkaListener` instead of a `@StreamListener`; `@KafkaNull` is handled by spring-kafka for `@KafkaListener`.

I decided to add another test for input bindings and found it needed more changes to the converter (and to the core project - see [here for the polishing commit](https://github.com/garyrussell/spring-cloud-stream-binder-kafka/commit/dae3b9cad5ca41128d4c861d02dc4b8d526dd0c5).

@aldex32 , @olegz, @artembilan please review.

{Core PR is [here](https://github.com/spring-cloud/spring-cloud-stream/pull/1602).

**Comment by aldex32** [comment]
@garyrussell LGTM

**Comment by garyrussell** [comment]
Ready to merge; just waiting a clean build of stream core; otherwise the test will fail.

**Comment by garyrussell** [comment]
Merged as 4d02c38f7060eadabe2b9e7ad3a6bc27086ea9e1 after adding code to support inbound tombstones too.

**Pull request closed without merging**